### DISCUSSION TITLE
Help! Help! Help!

### DISCUSSION THREAD

**Discussion by huboyang040911** [open-mmlab/mmaction2#2916] [Q&A]
我自己录制了视频并按照官方说明构造了自己的ava数据集，一共有两个动作类别（hit，sit），使用的配置文件是mmaction2/configs/detection/slowfast下的slowonly_kinetics400-pretrained-r50_8xb16-4x16x1-20e_ava21-rgb.py，但我在训练时遇到了问题：所有的mAP都是0，但损失是有在下降的，有人能帮帮我吗？
![Uploading 屏幕截图 2026-01-28 110914.png…]()

**Comment by huboyang040911** [comment]
训练日志如下
<img width="1090" height="625" alt="屏幕截图 2026-01-28 110914" src="https://github.com/user-attachments/assets/e03f77ec-c4bb-4460-86e1-0f1540669fb2" />

**Comment by huboyang040911** [comment]
I recorded the videos myself and built my own AVA dataset following the official guidelines, which includes two action categories (hit, sit). The configuration file I used is slowonly_kinetics400-pretrained-r50_8xb16-4x16x1-20e_ava21-rgb.py in the directory of mmaction2/configs/detection/slowfast. However, I ran into an issue during training: all mAP values are 0, while the loss is actually decreasing. Could anyone help me with this problem?
